pyiron/atomistics/structure/atoms.py

```diff
@@ -191,7 +192,16 @@ def __init__(
             self.dimension = len(self.positions[0])
         else:
             self.dimension = 0
-        self.visualize = Visualize(self)
+        self._visualize = Visualize(self)
+        self._analyse = Analyse(self)
```

Comment:
So this is not an issue with this PR per-se, as we already had `analyse_pyscal...`, but I see now that we mix American `Visualize` and British `Analyse`. In a future PR we should probably rename to pick just one.

Reply:
Yeah first I had `analyze` but then I saw the conflict with pyscal. Since `visualize` is not supposed to be used explicitly, we can change it to `visualise`. Otherwise we simply explain Brits prefer analytic stuff and North Americans prefer visual effects 😎.

Reply:
😝